source/constants.js

```diff
@@ -1,5 +1,6 @@
 exports.SAFE_DELIMITER = "→";
 exports.MAX_SET_TIMEOUT = 2 ** 31 - 1;
```

Comment:
Does this match max size 32 bit integer? This *looks* like that, so JS may already have a constant for it.

Reply:
You might be thinking of MAX_SAFE_INTEGER, which is 2^53 - 1